### PR TITLE
Mention custom scripts in mix release "One-off commands" docs

### PR DESCRIPTION
**Pull request by wojtekmach**
I think most projects will end up with some kind of `MyApp.ReleaseTasks` module so I think we could mention it in the docs.

### CODE REVIEW

File: lib/mix/lib/mix/tasks/release.ex [hunk 1]
```diff
@@ -123,6 +123,63 @@ defmodule Mix.Tasks.Release do
...
+      end
+
+  Again, for migrating the database we'd probably use `eval` but to list
+  currently connected users we'd probably connect to a running system using
+  `rpc`:
+
+      bin/RELEASE_NAME eval "MyApp.ReleaseTasks.migrate_db()"
+      bin/RELEASE_NAME rpc "MyApp.ReleaseTasks.print_connected_users()"
+
+  Finally, you may still automate this further by creating standalone scripts
```
Comment (wojtekmach) [inline lib/mix/lib/mix/tasks/release.ex]: The section below has probably way too much code so maybe it's better to show the functions but let users fill in the blanks, or skip this thing altogether?

Comment (josevalim) [reply lib/mix/lib/mix/tasks/release.ex]: Yeah, we should probably keep this focused. I have pushed some updates, let me know what you think.

**Comment by wojtekmach** [comment]
@josevalim looks good!

**Comment by whatyouhide** [comment]
Thanks @wojtekmach! 💟

**Pull request merged**